scenarios/arcaflow/ocp-chaos/README.md

```diff
@@ -0,0 +1,304 @@
+# OpenShift Shenanigans
```

Comment:
@mfleader https://github.com/krkn-chaos/krkn/tree/main/utils might be a good place to host the parallelization effort until it's integrated in krkn after which we need to add a pointer ro https://github.com/krkn-chaos/krkn/blob/main/docs/getting_started.md. Thoughts?